docs/userguide/configure-controllers.md

```diff
@@ -48,9 +48,9 @@ E.g. Specify a controller list:
 --controllers=cluster,clusterStatus,binding,xxx
 ```
 
-E.g. Disable some controllers:
+E.g. Disable some controllers and keep other default controllers turned on:
```

Comment:
```suggestion
E.g. Disable some controllers(remember to keep `*` if you want to keep the rest controllers in the default list):
```
How about this?

Reply:
lgtm

Reply:
great! then please add this to your commit. And we are ready to go.

Reply:
completed. @RainbowMango